addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,25 @@
+---Models in module 'website_livechat'---
+---Fields in module 'website_livechat'---
+website_livechat / mail.channel             / livechat_active (boolean)     : NEW
+website_livechat / mail.channel             / livechat_visitor_id (many2one): NEW relation: website.visitor
+website_livechat / website.visitor          / livechat_operator_id (many2one): NEW relation: res.partner, isfunction: function, stored
```

Comment:
I think you should fill this in a similar manner as https://github.com/OCA/OpenUpgrade/pull/2321/files#diff-731049fb90a841c7cf2ae3ec345b644ab76cffd65af083c32f9ac5dbbbba08b4R15

Reply:
OK, I see, thanks for the insight. No need of doing anything indeed, as it's a new model, so no records in it, and guessing them is not suitable.

Reply:
Thanks for your feedback.